build_openroad.sh

```diff
@@ -91,29 +92,29 @@ if [ -z ${OPENROAD_FLOW_NO_GIT_INIT+x} ]; then
 fi
 
 if [ ! -z ${OR_BRANCH+x} ]; then
-  echo "updating OpenROAD tool to the latest"
+  echo "[INFO FLOW-0005] Updating tools/OpenROAD to the latest"
```

Comment:
Messages 1-5 look to be already in use in `write_ref_sdc.tcl`

Reply:
Hmm they shouldn't have been, 0-2 have been in use for a long time in the build script. I will adjust.